deployments/helm-chart/README.md

```diff
@@ -23,12 +23,13 @@ This step is required if you're installing the chart using its sources. Addition
 
 1. Clone the Ingress controller repo:
     ```console
-    $ git clone https://github.com/nginxinc/kubernetes-ingress/
+    $ git clone https://github.com/nginxinc/kubernetes-ingress --branch v2.1.1
     ```
+    **Note**: If you wish to use the experimental repository (`edge`), remove the `--branch` flag and value.
```

Comment:
```suggestion
    **Note**: If you want to use the experimental repository (`edge`), remove the `--branch` flag and value.
```

Reply:
FYI: F5 style guide says not to use "desire" or "wish." The suggested alternatives are "want, need, require, or prefer." I went with "want."

https://docs.f5net.com/pages/viewpage.action?spaceKey=GSASKF5&title=F5+Technical+Style+Guide